src/pages/components/WhySysl/WhySysl.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import classnames from 'classnames';
+import styles from './styles.module.css';
+
+const badges = [
+    {
+        title: <>Guaranteed Consistency</>,
+        imageUrl: 'img/icon/garanteed_consistency.svg',
+        description: (<>Sysl is built for change. With Sysl, your specifications become the single source of truth. All the related system specifications, code, diagrams, documentations keep the same version - no more stale designs. Also, in unified good style.</>),
+    },
+    {
+        title: <>Quality  Built-in</>,
+        imageUrl: 'img/icon/Quality Built-in.svg',
+        description: (<>With Sysl, you can generate high-quality code, diagrams and documentation which align to style guide and best practices. Embed security in from the start.</>)
+    },
+    {
+        title: <>Flexible</>,
+        imageUrl: 'img/icon/Flexible.svg',
+        description: (<>Sysl's open-ended architecture makes it possible to generation code for any programming language, with out-of-the-box support for Java and Go.</>),
+    },
+]
+
+function WhySysl() {
+    return (
+        <React.Fragment>
+            <div className={styles.why__sysl__title}>
+                <h1 className={classnames('text--primary text--center', styles.deliver__title)}>Why Sysl</h1>
+            </div>
+            <div className={styles.why__sysl__curve}>
+            </div>
+            <div className={styles.why__sysl}>
+                <div className={styles.why__sysl__badge__group}>
+                    {badges && badges.length > 0 &&
+                        badges.map((badge, idx) => {
+                            return (
+                                <div className={styles.why__sysl__badge} key={idx}>
+                                    <img src={badge.imageUrl}></img>
+                                    <h1 className="text--center">{badge.title}</h1>
+                                    <span>{badge.description}</span>
+                                </div>
+                            )
+                        })
+                    }
+                </div>
+
+                <div className="row">
+                    <div className="col col--12">
+                        <table>
+                            <thead>
+                                <tr>
+                                    <th>Features &amp; Target Customers</th>
```

Comment:
Nit: "Target customers" is business speak. No one wants to think of themself as a "target" :). Use something like "user", "role" or "audience".

```suggestion
                                    <th>Features &amp; Users</th>
```